src/genie/libs/parser/iosxe/tests/ShowWirelessCtsSummary/cli/equal/golden_output_output.txt

```diff
@@ -0,0 +1,51 @@
+Local Mode CTS Configuration
```

Comment:
this is actual one output from device? just curious if this output is created from 2 outputs because I saw 2x `Local Mode CTS Configuration`.

Reply:
I think we are correct. I changed the golden_output_output.txt to a different example that only has one `Local Mode CTS Configuration` and one `Flex Mode CTS Configuration`. I think that is more accurate.